Directory.Build.props

```diff
@@ -1,10 +1,9 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>3.1.0</VersionPrefix>
+    <VersionPrefix>3.2.0</VersionPrefix>
     <LangVersion>10.0</LangVersion>
     <Nullable>enable</Nullable>
-    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
This appears to be the best way to cope with a netcoreapp3.1 target.

Reply:
That's fine, but cope how?

Reply:
Could you disable it on just that project instead?

Reply:
At one point earlier it was breaking on an older target for some reason. Disabling it on the specific project would've been a better idea, but it appears to work now…